build-config

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash +xe
-REPO_NAME=`pushd ${WORKSPACE}/build >/dev/null && git remote show origin -n | grep "Fetch URL:" | sed "s#^.*/\(.*\).git#\1#" && popd > /dev/null`
+if [ -z "${REPO_NAME}" ]; then
+  REPO_NAME=`pushd ${WORKSPACE}/build >/dev/null && git remote show origin -n | grep "Fetch URL:" | sed "s#^.*/\(.*\).git#\1#" && popd > /dev/null`
+fi
 if [ -e "${REPO_NAME}/sonar.properties.in" ]; then
```

Comment:
if find dependency PR, REPO_NAME will be generated in "pr_parser.py" as a repo name list.